x-pack/plugins/apm/server/lib/errors/distribution/__tests__/get_buckets.test.ts

```diff
@@ -30,7 +30,8 @@ describe('timeseriesFetcher', () => {
         end: 1528977600000,
         client: clientSpy,
         config: {
-          get: () => 'myIndex' as any
+          get: () => 'myIndex' as any,
+          has: () => true
```

Comment:
@Stacey-Gammon Do we need to mock this property if we never call it?

Reply:
It appears so @sqren:

<img width="507" alt="screen shot 2019-02-26 at 11 34 22 am" src="https://user-images.githubusercontent.com/16563603/53429369-83d53880-39ba-11e9-9f33-5c6a86ef7fc9.png">

unless I am missing something, I believe since the function expects the Config object, even if it's not used internally, because it's required on the config object, it has to exist.